xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -125,7 +125,14 @@ bool CVideoPlayerVideo::OpenStream(CDVDStreamInfo hint)
   if (hint.flags & AV_DISPOSITION_ATTACHED_PIC)
     return false;
   if (hint.extrasize == 0)
-    return false;
+  {
+    // codecs with no extradata
+    if (hint.codec != AV_CODEC_ID_MJPEG &&
```

Comment:
@peak3d what codecs do require extradata? maybe we should change this to a positive list